CONTRIBUTING.md

```diff
@@ -1,12 +1,18 @@
 # Contributing to rticonnext-examples
 
-### Contributor License Agreement (CLA)
-In order to accept your pull request, we need you to sign a Contributor License Agreement (CLA). Complete your CLA here: http://community.rti.com/cla. You only need to do this once, we cross-check your Github username with the list of contributors who have signed the CLA.
-
-### License
-RTI grants Licensee a license to use, modify, compile, and create derivative works 
-of the Software.  Licensee has the right to distribute object form only for use with RTI 
-products.  The Software is provided “as is”, with no warranty of any type, including 
-any warranty for fitness for any purpose. RTI is under no obligation to maintain or 
-support the Software.  RTI shall not be liable for any incidental or consequential 
-damages arising out of the use or inability to use the software.
+## Contributor License Agreement (CLA)
```

Comment:
@fgaranda it would be really nice to finally integrate https://www.clahub.com/ in the repository

Reply:
Sounds good.